.github/workflows/release.yml

```
@@ -0,0 +1,53 @@
+          
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+          
+      - name: Install dependencies
+        run: |
+          npm ci
+          npm install @biomejs/cli-linux-x64 @rollup/rollup-linux-x64-gnu @esbuild/linux-x64
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)